src/nvim/normal.c

```diff
@@ -3586,13 +3586,16 @@ void scroll_redraw(int up, long count)
  */
 static void nv_zet(cmdarg_T *cap)
 {
-  long n;
+  int n;
   colnr_T col;
   int nchar = cap->nchar;
   long old_fdl = curwin->w_p_fdl;
   int old_fen = curwin->w_p_fen;
   bool undo = false;
 
+  assert(p_siso <= INT_MAX);
+  int p_siso_l = (int)p_siso;
```

Comment:
Any reason not to prefix `l_` prefix as you usually do? I'd prefer to always use the same pattern.